chipsec/cfg/8086/icx.xml

```diff
@@ -0,0 +1,221 @@
+
+Contact information:
+chipsec@intel.com
+-->
+
+  <!-- #################################### -->
+  <!--                                      -->
+  <!-- Information                          -->
+  <!--                                      -->
+  <!-- #################################### -->
+  <info family="xeon" detection_value='606A5, 606A6'>
+    <sku did="0x09A2" name="IceLake" code="ICX" longname="Intel Xeon Processor E5/E7 v7 (Icelake CPU)" />
+  </info>
+
+  <pci>
+    <device name="MemMap_VTd" bus="0" dev="0"    fun="0" vid="0x8086" did="0x09A2" />
+    <device name="SATA"       bus="0" dev="0x17" fun="0" vid="0x8086" did="0xA182,0xA186,0xA202,0xA206,0x2822,0x2826,0x1C02" />
+    <device name="P2SBC"      bus="0" dev="0x1F" fun="1" vid="0x8086" did="0xA220,0xA1A0" />
+    <device name="PMC"        bus="0" dev="0x1F" fun="2" vid="0x8086" did="0xA221,0xA1A1" />
+    <device name="SMBUS"      bus="0" dev="0x1F" fun="4" vid="0x8086" did="0xA223,0xA1A3" />
+    <device name="SPI"        bus="0" dev="0x1F" fun="5" vid="0x8086" did="0xA224,0xA1A4" />
+  </pci>
+
+  <mmio>
+    <bar name="MMCFG"    bus="0" dev="0"    fun="0" reg="0x90" width="8" mask="0x7FFFFFC000000" size="0x1000" desc="PCI Express Register Range"/>
+    <bar name="DMIBAR"   bus="0" dev="3"    fun="0" reg="0x50" width="4" mask="0xFFFFF000"      size="0x1000" enable_bit="0" desc="Root Complex Register Range"/>
+    <bar name="PWRMBASE" bus="0" dev="0x1F" fun="2" reg="0x48" width="4" mask="0xFFFFF000"      size="0x1000" desc="PM Base Address"/>
+    <bar name="SPIBAR"   bus="0" dev="0x1F" fun="5" reg="0x10" width="4" mask="0xFFFFF000"      size="0x1000" desc="SPI Controller Register Range"/>
+    <bar name="TCO_BAR"  bus="0" dev="0x1F" fun="4" reg="0x50" width="4" mask="0xFFE0"          size="0x1000" desc="TCO Base Register Register Range"/>
+    <bar name="SBREGBAR" register="SBREG_BAR" base_field="RBA" size="0x1000000" desc="Sideband Register Access BAR"/>
+  </mmio>
+
+  <io>
+    <bar name="TCOBAR" register="TCO_BAR" base_field="Base" size="0x80" desc="TCO Base Address"/>
+  </io>
+
+  <memory>
+  </memory>
+
+  <registers>
+
+    <!-- PCH -->
+    <register name="ABAR"    type="pcicfg" device="SATA" offset="0x24" size="4" desc="AHCI Base Address">
+      <field name="BA" bit="19" size="13" desc="Base Address"/>
+    </register>
+    <register name="SATAGC"  type="pcicfg" device="SATA" offset="0x9C" size="4" desc="SATA General Configuration">
+      <field name="ASSEL" bit="0" size="3" desc="ABAR Size Select"/>
+    </register>
+
+    <!-- Sideband Register Access Registers -->
+    <register name="SBREG_BAR"  type="pcicfg" device="P2SBC" offset="0x10" size="4" desc="Sideband Register Access BAR">
+      <field name="RBA" bit="24" size="8" desc="Register Base Address"/>
+    </register>
+    <register name="SBREG_BARH" type="pcicfg" device="P2SBC" offset="0x14" size="4" desc="Sideband Register Access BAR High DWORD">

```

Comment:
Shouldn't this be combined with SBREG_BAR?

Reply:
Removed duplicated PCH